be/src/storage/task/engine_clone_task.cpp

```diff
@@ -209,6 +209,13 @@ Status EngineCloneTask::_do_clone_primary_tablet(Tablet* tablet) {
     vector<int64_t> missing_version_ranges;
     st = tablet->updates()->get_missing_version_ranges(missing_version_ranges);
     if (st.ok()) {
+        // Probably no missing version in this case, we just return FE the clone success to avoid wasted work.
```

Comment:
Why not change the logic in get_missing_version_ranges()? This is too trick.

Reply:
yes, we better change `get_missing_version_ranges()`